scripts/start-streamlit-app-linux

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Could you try if this script works in your environment? If it doesn't could you lemme know?